CHANGELOG.md

```diff
@@ -0,0 +1,53 @@
+# Changelog
+
+#### v4.4.0
```

Comment:
I noticed that the package is actually on [v5.2](https://github.com/nmn/react-timeago/blob/master/package.json#L3) - so I bumped this here. Is that desired?